packages/react/src/SelectPanel/SelectPanel.docs.json

```diff
@@ -127,6 +127,12 @@
       "description": "Callback when the search input changes",
       "defaultValue": ""
     },
+    {
+      "name": "onCancel",
+      "type": "() => void",
+      "description": "(Narrow screens) Callback when the user hits cancel or close",
```

Comment:
do we expect this to be used in modal as well? If so maybe we need to correct this verbiage

Reply:
We do, but we don't have that yet. Can update once we have modal.

It's going to be a strange description: `Narrow screens or variant=modal` 😅